README.md

```diff
@@ -625,6 +633,10 @@ ec2ShareAmi(
 
 ## current master
 
+## 1.34
+* add support for assume role with SAML assertion authentication (#140)
+* add withRole step to assume different role with a same AWS session
```

Comment:
Can you remove this, please.

Reply:
Everything or juste the last line ?
If everything, please update the pr template, because it's ask to mention changes in the changelog.

Reply:
Sorry for the confusion. Thank you for writing a changelog at all. I just meant the line for `withRole`.